### PR TITLE
Ui improvements and new fields

### PR DESCRIPTION
**Pull request by jotacervan**
It is required the https://github.com/jotacervan/bookstore-api to work.

Make sure you have the api application running at the 3000 port.
**Installation**

* `cd bookstore`
* `npm install`

**Running**
* `ember server --proxy http://localhost:3000`

*obs: It is important to start the api application before running the ember server.*

Features:
1. Now the application parses the biography to html when it find markup language on it.
2. It was recreated the layout with ember-paper ( Page, Container, purchase modal ).

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+trim_trailing_whitespace = true
+insert_final_newline = true
+indent_style = space
+indent_size = 2
+
+[*.hbs]
+insert_final_newline = false
+
+[*.{diff,md}]
+trim_trailing_whitespace = false
```
Comment (lucasprag) [inline .editorconfig:20]: yes yes yes

**Review by lucasprag — COMMENTED** [review]
Great job on this project @jotacervan 🎉 

Could you add some screenshots please? Thanks

**Comment by jotacervan** [comment]
Thanks @lucasprag, Follow the screenshots below.

Screen Shot 2019-12-18 at 2 24 06 PM


Screen Shot 2019-12-18 at 2 24 15 PM


Screen Shot 2019-12-18 at 2 24 32 PM

Screen Shot 2019-12-18 at 2 24 43 PM

**Review by lucasprag — APPROVED** [review]
Great work! You can merge this PR now and this wraps-up the front-end part of the test 👍 🎉

**Pull request merged**